packages/flutter/lib/src/material/circle_avatar.dart

```diff
@@ -57,6 +58,8 @@ class CircleAvatar extends StatelessWidget {
     this.radius,
     this.minRadius,
     this.maxRadius,
+    this.borderColor = Colors.transparent,
```

Comment:
until we have a lint that checks this, please use colons for default argument syntax for consistency

Reply:
Oh, I didn't know that was a thing. Will do.